__tests__/contact/contact.mutation.test.js

```diff
@@ -1,215 +1,196 @@
-      .mutate({
-        mutation: gql`
-          mutation($id: ID!, $contact: contactInput!) {
-            updateContact(id: $id, contact: $contact) {
-              ... on Contact {
-                _id
-                phoneNumber
-                openHours {
-                  lang
-                  value
-                }
-                address {
-                  lang
-                  value
-                }
-                email
-                images {
-                  value {
-                    medium
-                  }
-                }
-                link
+  it('#2 should update contact', async () => {
+    const res = await operations.mutate({
+      mutation: gql`
+        mutation($id: ID!, $contact: contactInput!) {
+          updateContact(id: $id, contact: $contact) {
+            ... on Contact {
+              _id
+              phoneNumber
+              openHours {
+                lang
+                value
               }
-              ... on Error {
-                message
-                statusCode
+              address {
+                lang
+                value
               }
+              email
+              images {
+                value {
+                  medium
+                }
+              }
+              link
+            }
+            ... on Error {
+              message
+              statusCode
             }
           }
-        `,
-        variables: { id: contactsId, contact: updatedContact },
-      })
-      .catch(e => e);
-    expect(res.data.updateContact).toHaveProperty(
-      'email',
-      updatedContact.email
-    );
-    expect(res.data.updateContact.images).toBeInstanceOf(Array);
-    expect(res.data.updateContact).toHaveProperty('images', [
+        }
+      `,
+      variables: {
+        id: contactsId,
+        contact: updatedContact,
+      },
+    });
+
+    const updatedContactRes = res.data.updateContact;
+
+    expect(updatedContactRes).toHaveProperty('email', updatedContact.email);
+
+    expect(updatedContactRes).toHaveProperty('images', [
       {
-        __typename: 'LanguageImageSet',
         value: {
-          __typename: 'ImageSet',
           medium: updatedContact.images[0].value.medium,
         },
       },
       {
-        __typename: 'LanguageImageSet',
         value: {
-          __typename: 'ImageSet',
           medium: updatedContact.images[1].value.medium,
         },
       },
     ]);
-    expect(res.data.updateContact).toHaveProperty('address', [
-      {
-        __typename: 'Language',
-        lang: updatedContact.address[0].lang,
-        value: updatedContact.address[0].value,
-      },
-      {
-        __typename: 'Language',
-        lang: updatedContact.address[1].lang,
-        value: updatedContact.address[1].value,
-      },
-    ]);
+
+    expect(updatedContactRes.images).toBeInstanceOf(Array);
+    expect(updatedContactRes).toHaveProperty(
+      'address',
+      updatedContact.address.map(item => ({
```

Comment:
Please, simplify it to the `contact.openHours` because mapping is useless now due to the returning the same object